scheduler.go

```diff
@@ -147,15 +150,29 @@ func (sc *Scheduler) runTask(ctx context.Context, running *int32, task Task) err
 func (sc *Scheduler) execService(ctx context.Context, task Task) error {
 	execID, err := sc.client.ContainerExecCreate(ctx, task.Container, types.ExecConfig{
 		Cmd: task.Command,
+		AttachStderr:true,
+		AttachStdout:true,
 	})
 	if err != nil {
 		return fmt.Errorf("create exec for %s: %w", task.Service, err)
 	}
 
-	err = sc.client.ContainerExecStart(ctx, execID.ID, types.ExecStartCheck{})
+	attach, err := sc.client.ContainerExecAttach(ctx, execID.ID, types.ExecStartCheck{})
 	if err != nil {
-		return fmt.Errorf("exec for %s: %w", task.Service, err)
+	    return fmt.Errorf("exec for %s: %w", task.Service, err)
 	}
+	defer attach.Close()
+	if (task.logging) {
+	    io.Copy(log.Writer(), attach.Reader)
+    }
+
+	inspect, err := sc.client.ContainerExecInspect(ctx, execID.ID)
+    if err != nil {
+    	return fmt.Errorf("inspect exec for %s: %w", task.Service, err)
+    }
+    if inspect.ExitCode != 0 {
+    	return fmt.Errorf("command returned non-zero code %d", inspect.ExitCode)
+    }
```

Comment:
let's just copy-paste-split function and call it `execAttachService` and `execStartService`. In `execService` just route to old function (`execStartService`) and to the new one.

Reply:
Ok I think I got you... gonna make new commit now